inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/CreateSortConfigListenerV2.java

```diff
@@ -91,6 +92,11 @@ public ListenerResult listen(WorkflowContext context) throws Exception {
         InlongGroupInfo groupInfo = form.getGroupInfo();
         List<InlongStreamInfo> streamInfos = form.getStreamInfos();
         final String groupId = groupInfo.getInlongGroupId();
+        List<StreamSink> streamSinks = sinkService.listSink(groupId, null);
+        if (CollectionUtils.isEmpty(streamSinks)) {
+            log.info("No sinks configured, assuming direct consumption from MQ");
```

Comment:
Suggest changing the log level to warn, and changing the log info to `not any sinks found for groupId={}, skip to create sort config`.